src/main/scala/util/Annotations.scala

```diff
@@ -261,7 +261,7 @@ object GenRegDescsAnno {
 
     val moduleName = rawModule.name
     val baseHex = s"0x${baseAddress.toInt.toHexString}"
-    val displayName = s"${moduleName}.${baseHex}"
+    s"${moduleName}.${baseHex}"
```

Comment:
Seems like this one might be an actual bug, that this isn't passed to `RegistersSer.displayName` below? @derekpappas

Reply:
The removed `displayName` isn't referenced in the local scope

Reply:
I meant that it should be referenced in the local scope and it is a bug in the local scope that it is not